packages/core/src/blocks/core/group/index.js

```diff
@@ -88,17 +90,53 @@ const addEditorBlockAttributes = createHigherOrderComponent( ( BlockListBlock )
 
 const applyFrontEndClasses = ( extraProps, blockType, attributes ) =>{
```

Comment:
This filter should sticky to altering the `className` prop. Adding the CSS custom properties to the style prop should be done in the spaceAndSizing component based on a property through the `supports` API